Adds more fire alarms to the center of the Diogaras

Daylight2 opened the pull request:
<!-- By ticking or leaving ticked the option "Allow edits and access to secrets by maintainers" you give permission for repository maintainers to push changes to your branch without explicitly asking. -->

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do

Adds a few fire alarms to the central area of emeraldstation.
<!-- Include a small to medium description of what your PR changes. -->
<!-- Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
<!-- If your PR fixes an issue, add "Fixes #1234" somewhere in the PR description. This will automatically close the bug upon PR submission. -->

## Why It's Good For The Game

This massive zone had a total of one fire alarm, with most fire zones having no fire alarm within them. Compared to other hallways, where every area sealed with fire doors has at least one fire alarm. 
<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes

![Screenshot 2025-01-30 051442](https://github.com/user-attachments/assets/cdc47ccc-c66a-492d-ac8d-8888aaab1c7c)


## Testing

Pressed the fire alarms.
<!-- How did you test the PR, if at all? -->

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  <!-- Replace the box with [x] to mark as complete. -->
  <!-- Ensure there are no spaces between the x and the square brackets [] else this will not work properly. -->
<hr>

## Changelog

:cl:
add: Added more fire alarms to the center of the Diogaras
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

warriorstar-orion reviewed (changes requested):
These three alarms are a little close together. Could you move the middle one where the APC is and move the APC up a tile, and move the right air alarm to the north wall?
![image](https://github.com/user-attachments/assets/f3b9edba-de2f-4718-8ecd-0dcbbc6a2472)

Daylight2 commented:
> These three alarms are a little close together. Could you move the middle one where the APC is and move the APC up a tile, and move the right air alarm to the north wall? ![image](https://private-user-images.githubusercontent.com/59303604/407976616-

Looking at the rest of the station, it's quite rare for fire alarms to be right next to firelocks, even when there's room / seemingly good reason. To that affect, I moved the alarms around a bit and removed one from the south hallway (map image updated on PR description)
